lib/models/food.js

```diff
@@ -25,13 +32,20 @@ function destroy(req) {
   return database.raw('DELETE FROM foods WHERE id = ?', req.params.id)
 }
 
-function validation(food, callback) {
-  if (!food.calories) {
+function calorieValidation(calories, callback) {
+  if (!calories) {
     return callback(new Error('Food must have calories.'));
   }
   callback();
 }
 
+function nameValidation(name, callback) {
+  if (!name) {
+    return callback(new Error('Food must have name.'));
+  }
+  callback();
+}
+
```

Comment:
🎉 🎉 I think that having these in the backend is going to make implementing the front end go smoothly.